### ISSUE TITLE
[Craft Scheduler Thread - 2/WARN]: Exception in thread "Craft Scheduler Thread - 2"

### ISSUE THREAD

**Issue by KD148** [CraftG/WatchCat#6]
WatchCat | Machine Learning AntiCheat 1.6.2
Java 8
ProtocolLib v4.3.0
PaperSpigot-1.8.8-R0.1-SNAPSHOT-latest.jar

Pls Fix & that the random player is not visible in the tab bar

```
[16:39:04] [Craft Scheduler Thread - 2/WARN]: Exception in thread "Craft Scheduler Thread - 2" 
[16:39:04] [Craft Scheduler Thread - 2/WARN]: org.apache.commons.lang.UnhandledException: Plugin WatchCat v1.6.2 generated an exception while executing task 36
	at org.bukkit.craftbukkit.v1_8_R3.scheduler.CraftAsyncTask.run(CraftAsyncTask.java:56)
	at org.github.paperspigot.ServerSchedulerReportingWrapper.run(ServerSchedulerReportingWrapper.java:23)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1149)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:624)
	at java.lang.Thread.run(Thread.java:748)
Caused by: java.lang.NullPointerException
	at xyz.WatchCat.api.WatchCatAPI.addVL(Unknown Source)
	at xyz.WatchCat.IIIiIiiIii.run(Unknown Source)
	at org.bukkit.craftbukkit.v1_8_R3.scheduler.CraftTask.run(CraftTask.java:59)
	at org.bukkit.craftbukkit.v1_8_R3.scheduler.CraftAsyncTask.run(CraftAsyncTask.java:53)
	... 4 more

[16:39:04] [Craft Scheduler Thread - 9/WARN]: Exception in thread "Craft Scheduler Thread - 9" 
[16:39:04] [Craft Scheduler Thread - 9/WARN]: org.apache.commons.lang.UnhandledException: Plugin WatchCat v1.6.2 generated an exception while executing task 36
	at org.bukkit.craftbukkit.v1_8_R3.scheduler.CraftAsyncTask.run(CraftAsyncTask.java:56)
	at org.github.paperspigot.ServerSchedulerReportingWrapper.run(ServerSchedulerReportingWrapper.java:23)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1149)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:624)
	at java.lang.Thread.run(Thread.java:748)
Caused by: java.lang.NullPointerException
	at xyz.WatchCat.api.WatchCatAPI.addVL(Unknown Source)
	at xyz.WatchCat.IIIiIiiIii.run(Unknown Source)
	at org.bukkit.craftbukkit.v1_8_R3.scheduler.CraftTask.run(CraftTask.java:59)
	at org.bukkit.craftbukkit.v1_8_R3.scheduler.CraftAsyncTask.run(CraftAsyncTask.java:53)
	... 4 more

[16:39:04] [Craft Scheduler Thread - 10/WARN]: Exception in thread "Craft Scheduler Thread - 10" 
[16:39:04] [Craft Scheduler Thread - 10/WARN]: org.apache.commons.lang.UnhandledException: Plugin WatchCat v1.6.2 generated an exception while executing task 36
	at org.bukkit.craftbukkit.v1_8_R3.scheduler.CraftAsyncTask.run(CraftAsyncTask.java:56)
	at org.github.paperspigot.ServerSchedulerReportingWrapper.run(ServerSchedulerReportingWrapper.java:23)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1149)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:624)
	at java.lang.Thread.run(Thread.java:748)
Caused by: java.lang.NullPointerException
	at xyz.WatchCat.api.WatchCatAPI.addVL(Unknown Source)
	at xyz.WatchCat.IIIiIiiIii.run(Unknown Source)
	at org.bukkit.craftbukkit.v1_8_R3.scheduler.CraftTask.run(CraftTask.java:59)
	at org.bukkit.craftbukkit.v1_8_R3.scheduler.CraftAsyncTask.run(CraftAsyncTask.java:53)
	... 4 more

[16:39:04] [Craft Scheduler Thread - 12/WARN]: Exception in thread "Craft Scheduler Thread - 12" 
[16:39:04] [Craft Scheduler Thread - 12/WARN]: org.apache.commons.lang.UnhandledException: Plugin WatchCat v1.6.2 generated an exception while executing task 36
	at org.bukkit.craftbukkit.v1_8_R3.scheduler.CraftAsyncTask.run(CraftAsyncTask.java:56)
	at org.github.paperspigot.ServerSchedulerReportingWrapper.run(ServerSchedulerReportingWrapper.java:23)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1149)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:624)
	at java.lang.Thread.run(Thread.java:748)
Caused by: java.lang.NullPointerException
	at xyz.WatchCat.api.WatchCatAPI.addVL(Unknown Source)
	at xyz.WatchCat.IIIiIiiIii.run(Unknown Source)
	at org.bukkit.craftbukkit.v1_8_R3.scheduler.CraftTask.run(CraftTask.java:59)
	at org.bukkit.craftbukkit.v1_8_R3.scheduler.CraftAsyncTask.run(CraftAsyncTask.java:53)
	... 4 more

[16:39:04] [Craft Scheduler Thread - 13/WARN]: Exception in thread "Craft Scheduler Thread - 13" 
[16:39:04] [Craft Scheduler Thread - 13/WARN]: org.apache.commons.lang.UnhandledException: Plugin WatchCat v1.6.2 generated an exception while executing task 36
	at org.bukkit.craftbukkit.v1_8_R3.scheduler.CraftAsyncTask.run(CraftAsyncTask.java:56)
	at org.github.paperspigot.ServerSchedulerReportingWrapper.run(ServerSchedulerReportingWrapper.java:23)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1149)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:624)
	at java.lang.Thread.run(Thread.java:748)
Caused by: java.lang.NullPointerException
	at xyz.WatchCat.api.WatchCatAPI.addVL(Unknown Source)
	at xyz.WatchCat.IIIiIiiIii.run(Unknown Source)
	at org.bukkit.craftbukkit.v1_8_R3.scheduler.CraftTask.run(CraftTask.java:59)
	at org.bukkit.craftbukkit.v1_8_R3.scheduler.CraftAsyncTask.run(CraftAsyncTask.java:53)
	... 4 more

[16:39:04] [Craft Scheduler Thread - 11/WARN]: Exception in thread "Craft Scheduler Thread - 11" 
[16:39:04] [Craft Scheduler Thread - 11/WARN]: org.apache.commons.lang.UnhandledException: Plugin WatchCat v1.6.2 generated an exception while executing task 36
	at org.bukkit.craftbukkit.v1_8_R3.scheduler.CraftAsyncTask.run(CraftAsyncTask.java:56)
	at org.github.paperspigot.ServerSchedulerReportingWrapper.run(ServerSchedulerReportingWrapper.java:23)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1149)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:624)
	at java.lang.Thread.run(Thread.java:748)
Caused by: java.lang.NullPointerException
	at xyz.WatchCat.api.WatchCatAPI.addVL(Unknown Source)
	at xyz.WatchCat.IIIiIiiIii.run(Unknown Source)
	at org.bukkit.craftbukkit.v1_8_R3.scheduler.CraftTask.run(CraftTask.java:59)
	at org.bukkit.craftbukkit.v1_8_R3.scheduler.CraftAsyncTask.run(CraftAsyncTask.java:53)
	... 4 more

[16:39:05] [Craft Scheduler Thread - 14/WARN]: Exception in thread "Craft Scheduler Thread - 14" 
[16:39:05] [Craft Scheduler Thread - 14/WARN]: org.apache.commons.lang.UnhandledException: Plugin WatchCat v1.6.2 generated an exception while executing task 36
	at org.bukkit.craftbukkit.v1_8_R3.scheduler.CraftAsyncTask.run(CraftAsyncTask.java:56)
	at org.github.paperspigot.ServerSchedulerReportingWrapper.run(ServerSchedulerReportingWrapper.java:23)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1149)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:624)
	at java.lang.Thread.run(Thread.java:748)
Caused by: java.lang.NullPointerException
	at xyz.WatchCat.api.WatchCatAPI.addVL(Unknown Source)
	at xyz.WatchCat.IIIiIiiIii.run(Unknown Source)
	at org.bukkit.craftbukkit.v1_8_R3.scheduler.CraftTask.run(CraftTask.java:59)
	at org.bukkit.craftbukkit.v1_8_R3.scheduler.CraftAsyncTask.run(CraftAsyncTask.java:53)
	... 4 more

[16:39:05] [Craft Scheduler Thread - 16/WARN]: Exception in thread "Craft Scheduler Thread - 16" 
[16:39:05] [Craft Scheduler Thread - 16/WARN]: org.apache.commons.lang.UnhandledException: Plugin WatchCat v1.6.2 generated an exception while executing task 36
	at org.bukkit.craftbukkit.v1_8_R3.scheduler.CraftAsyncTask.run(CraftAsyncTask.java:56)
	at org.github.paperspigot.ServerSchedulerReportingWrapper.run(ServerSchedulerReportingWrapper.java:23)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1149)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:624)
	at java.lang.Thread.run(Thread.java:748)
Caused by: java.lang.NullPointerException
	at xyz.WatchCat.api.WatchCatAPI.addVL(Unknown Source)
	at xyz.WatchCat.IIIiIiiIii.run(Unknown Source)
	at org.bukkit.craftbukkit.v1_8_R3.scheduler.CraftTask.run(CraftTask.java:59)
	at org.bukkit.craftbukkit.v1_8_R3.scheduler.CraftAsyncTask.run(CraftAsyncTask.java:53)
	... 4 more

[16:39:05] [Craft Scheduler Thread - 17/WARN]: Exception in thread "Craft Scheduler Thread - 17" 
[16:39:05] [Craft Scheduler Thread - 17/WARN]: org.apache.commons.lang.UnhandledException: Plugin WatchCat v1.6.2 generated an exception while executing task 36
	at org.bukkit.craftbukkit.v1_8_R3.scheduler.CraftAsyncTask.run(CraftAsyncTask.java:56)
	at org.github.paperspigot.ServerSchedulerReportingWrapper.run(ServerSchedulerReportingWrapper.java:23)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1149)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:624)
	at java.lang.Thread.run(Thread.java:748)
Caused by: java.lang.NullPointerException
	at xyz.WatchCat.api.WatchCatAPI.addVL(Unknown Source)
	at xyz.WatchCat.IIIiIiiIii.run(Unknown Source)
	at org.bukkit.craftbukkit.v1_8_R3.scheduler.CraftTask.run(CraftTask.java:59)
	at org.bukkit.craftbukkit.v1_8_R3.scheduler.CraftAsyncTask.run(CraftAsyncTask.java:53)
	... 4 more

[16:39:05] [Craft Scheduler Thread - 15/WARN]: Exception in thread "Craft Scheduler Thread - 15" 
[16:39:05] [Craft Scheduler Thread - 15/WARN]: org.apache.commons.lang.UnhandledException: Plugin WatchCat v1.6.2 generated an exception while executing task 36
	at org.bukkit.craftbukkit.v1_8_R3.scheduler.CraftAsyncTask.run(CraftAsyncTask.java:56)
	at org.github.paperspigot.ServerSchedulerReportingWrapper.run(ServerSchedulerReportingWrapper.java:23)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1149)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:624)
	at java.lang.Thread.run(Thread.java:748)
Caused by: java.lang.NullPointerException
	at xyz.WatchCat.api.WatchCatAPI.addVL(Unknown Source)
	at xyz.WatchCat.IIIiIiiIii.run(Unknown Source)
	at org.bukkit.craftbukkit.v1_8_R3.scheduler.CraftTask.run(CraftTask.java:59)
	at org.bukkit.craftbukkit.v1_8_R3.scheduler.CraftAsyncTask.run(CraftAsyncTask.java:53)
	... 4 more

[16:39:05] [Craft Scheduler Thread - 19/WARN]: Exception in thread "Craft Scheduler Thread - 19" 
[16:39:05] [Craft Scheduler Thread - 19/WARN]: org.apache.commons.lang.UnhandledException: Plugin WatchCat v1.6.2 generated an exception while executing task 36
	at org.bukkit.craftbukkit.v1_8_R3.scheduler.CraftAsyncTask.run(CraftAsyncTask.java:56)
	at org.github.paperspigot.ServerSchedulerReportingWrapper.run(ServerSchedulerReportingWrapper.java:23)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1149)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:624)
	at java.lang.Thread.run(Thread.java:748)
Caused by: java.lang.NullPointerException
	at xyz.WatchCat.api.WatchCatAPI.addVL(Unknown Source)
	at xyz.WatchCat.IIIiIiiIii.run(Unknown Source)
	at org.bukkit.craftbukkit.v1_8_R3.scheduler.CraftTask.run(CraftTask.java:59)
	at org.bukkit.craftbukkit.v1_8_R3.scheduler.CraftAsyncTask.run(CraftAsyncTask.java:53)
	... 4 more

[16:39:05] [Craft Scheduler Thread - 20/WARN]: Exception in thread "Craft Scheduler Thread - 20" 
[16:39:05] [Craft Scheduler Thread - 20/WARN]: org.apache.commons.lang.UnhandledException: Plugin WatchCat v1.6.2 generated an exception while executing task 36
	at org.bukkit.craftbukkit.v1_8_R3.scheduler.CraftAsyncTask.run(CraftAsyncTask.java:56)
	at org.github.paperspigot.ServerSchedulerReportingWrapper.run(ServerSchedulerReportingWrapper.java:23)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1149)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:624)
	at java.lang.Thread.run(Thread.java:748)
Caused by: java.lang.NullPointerException
	at xyz.WatchCat.api.WatchCatAPI.addVL(Unknown Source)
	at xyz.WatchCat.IIIiIiiIii.run(Unknown Source)
	at org.bukkit.craftbukkit.v1_8_R3.scheduler.CraftTask.run(CraftTask.java:59)
	at org.bukkit.craftbukkit.v1_8_R3.scheduler.CraftAsyncTask.run(CraftAsyncTask.java:53)
	... 4 more

[16:39:05] [Craft Scheduler Thread - 18/WARN]: Exception in thread "Craft Scheduler Thread - 18" 
[16:39:05] [Craft Scheduler Thread - 18/WARN]: org.apache.commons.lang.UnhandledException: Plugin WatchCat v1.6.2 generated an exception while executing task 36
	at org.bukkit.craftbukkit.v1_8_R3.scheduler.CraftAsyncTask.run(CraftAsyncTask.java:56)
	at org.github.paperspigot.ServerSchedulerReportingWrapper.run(ServerSchedulerReportingWrapper.java:23)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1149)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:624)
	at java.lang.Thread.run(Thread.java:748)
Caused by: java.lang.NullPointerException
	at xyz.WatchCat.api.WatchCatAPI.addVL(Unknown Source)
	at xyz.WatchCat.IIIiIiiIii.run(Unknown Source)
	at org.bukkit.craftbukkit.v1_8_R3.scheduler.CraftTask.run(CraftTask.java:59)
	at org.bukkit.craftbukkit.v1_8_R3.scheduler.CraftAsyncTask.run(CraftAsyncTask.java:53)
	... 4 more

[16:39:06] [Craft Scheduler Thread - 22/WARN]: Exception in thread "Craft Scheduler Thread - 22" 
[16:39:06] [Craft Scheduler Thread - 22/WARN]: org.apache.commons.lang.UnhandledException: Plugin WatchCat v1.6.2 generated an exception while executing task 36
	at org.bukkit.craftbukkit.v1_8_R3.scheduler.CraftAsyncTask.run(CraftAsyncTask.java:56)
	at org.github.paperspigot.ServerSchedulerReportingWrapper.run(ServerSchedulerReportingWrapper.java:23)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1149)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:624)
	at java.lang.Thread.run(Thread.java:748)
Caused by: java.lang.NullPointerException
	at xyz.WatchCat.api.WatchCatAPI.addVL(Unknown Source)
	at xyz.WatchCat.IIIiIiiIii.run(Unknown Source)
	at org.bukkit.craftbukkit.v1_8_R3.scheduler.CraftTask.run(CraftTask.java:59)
	at org.bukkit.craftbukkit.v1_8_R3.scheduler.CraftAsyncTask.run(CraftAsyncTask.java:53)
	... 4 more

[16:39:06] [Craft Scheduler Thread - 21/WARN]: Exception in thread "Craft Scheduler Thread - 21" 
[16:39:06] [Craft Scheduler Thread - 21/WARN]: org.apache.commons.lang.UnhandledException: Plugin WatchCat v1.6.2 generated an exception while executing task 36
	at org.bukkit.craftbukkit.v1_8_R3.scheduler.CraftAsyncTask.run(CraftAsyncTask.java:56)
	at org.github.paperspigot.ServerSchedulerReportingWrapper.run(ServerSchedulerReportingWrapper.java:23)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1149)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:624)
	at java.lang.Thread.run(Thread.java:748)
Caused by: java.lang.NullPointerException
	at xyz.WatchCat.api.WatchCatAPI.addVL(Unknown Source)
	at xyz.WatchCat.IIIiIiiIii.run(Unknown Source)
	at org.bukkit.craftbukkit.v1_8_R3.scheduler.CraftTask.run(CraftTask.java:59)
	at org.bukkit.craftbukkit.v1_8_R3.scheduler.CraftAsyncTask.run(CraftAsyncTask.java:53)
	... 4 more

[16:39:06] [Craft Scheduler Thread - 24/WARN]: Exception in thread "Craft Scheduler Thread - 24" 
[16:39:06] [Craft Scheduler Thread - 24/WARN]: org.apache.commons.lang.UnhandledException: Plugin WatchCat v1.6.2 generated an exception while executing task 36
	at org.bukkit.craftbukkit.v1_8_R3.scheduler.CraftAsyncTask.run(CraftAsyncTask.java:56)
	at org.github.paperspigot.ServerSchedulerReportingWrapper.run(ServerSchedulerReportingWrapper.java:23)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1149)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:624)
	at java.lang.Thread.run(Thread.java:748)
Caused by: java.lang.NullPointerException
	at xyz.WatchCat.api.WatchCatAPI.addVL(Unknown Source)
	at xyz.WatchCat.IIIiIiiIii.run(Unknown Source)
	at org.bukkit.craftbukkit.v1_8_R3.scheduler.CraftTask.run(CraftTask.java:59)
	at org.bukkit.craftbukkit.v1_8_R3.scheduler.CraftAsyncTask.run(CraftAsyncTask.java:53)
	... 4 more
```

**Comment by CraftG** [comment]
These errors have been fixed in version 1.6.2.1 :)

**Comment by CraftG** [comment]
These errors have been fixed in version 1.6.2.1 :)
